### PR TITLE
[i18n-ur] Added README_ur.md file

### PR DESCRIPTION
**Pull request by akkefa**
# What does this PR do?

Translate docs into Urdu language issue #33139

Documentation:
 @stevhliu @LysandreJik

### CODE REVIEW

**Comment by akkefa** [comment]
@stevhliu I’ve created a new pull request and hope to avoid any CircleCI errors this time.

**Comment by stevhliu** [comment]
@LysandreJik or @ydshieh, would you happen to know why the other two tests aren't being run?

**Comment by LysandreJik** [comment]
Given the changes only affect docs, I believe we can safely ignore the circle ci docs @stevhliu

File: README.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ limitations under the License.
         <a href="https://github.com/huggingface/transformers/blob/main/i18n/README_fr.md">Français</a> |
         <a href="https://github.com/huggingface/transformers/blob/main/i18n/README_de.md">Deutsch</a> |
         <a href="https://github.com/huggingface/transformers/blob/main/i18n/README_vi.md">Tiếng Việt</a> |
-	<a href="https://github.com/huggingface/transformers/blob/main/i18n/README_ar.md">العربية</a> |
+	    <a href="https://github.com/huggingface/transformers/blob/main/i18n/README_ar.md">العربية</a> |
+        <a href="https://github.com/huggingface/transformers/blob/main/i18n/README_ur.md">اردو</a> |
```
Comment (stevhliu) [inline README.md:52]: This looks misaligned

```suggestion
	<a href="https://github.com/huggingface/transformers/blob/main/i18n/README_ar.md">العربية</a> |
        <a href="https://github.com/huggingface/transformers/blob/main/i18n/README_ur.md">اردو</a> |
```

**Review by stevhliu — APPROVED** [review]
Thanks again!

**Comment by akkefa** [comment]
@stevhliu Who will merge the pull request ?

**Comment by ydshieh** [comment]
For the recorder

> The CI is running under akkefa personal account. Please follow https://support.circleci.com/hc/en-us/articles/360008097173-Troubleshooting-why-pull-requests-are-not-triggering-jobs-on-my-organization- to fix it.

> If you're following the fork instead of the upstream repo

A user who submits a pull request to your repository from a fork, but no pipeline is triggered with the pull request. This can happen when the user is following the project fork on their personal account rather than the project itself on CircleCI.

This will cause the jobs to trigger under the user's personal account. If the user is following a fork of the repository on CircleCI, we will only build on that fork and not the parent, so the parent’s PR will not get status updates. 

In these cases, the user unfollows their fork of the project on CircleCI. This will trigger their jobs to run under the organization when they submit pull requests. Those users can optionally follow the source project if they wish to see the pipelines.

**Pull request merged**